DependencyInjection/FOSRestExtension.php

```diff
@@ -171,6 +171,10 @@ private function loadFormatListener(array $config, XmlFileLoader $loader, Contai
                     'fos_rest.format_listener.media_type.version_regex',
                     $config['format_listener']['media_type']['version_regex']
                 );
+                $container->setParameter(
+                    'fos_rest.format_listener.media_type.default_version',
+                    $config['format_listener']['media_type']['default_version']
+                );
```

Comment:
I would do something like the following instead:

```php
$container->getDefinition('fos_rest.version_listener')->addArgument($config['format_listener']['media_type']['default_version']);
```

This way you don't have to worry about maintaining container parameters in the future (people might use them).